### PR TITLE
Read image data from path in case "image_path" is present

### PR DESCRIPTION
**Pull request by rom1504**
In metadata endpoint

### CODE REVIEW

**Comment by rom1504** [comment]
#170

File: clip_retrieval/clip_back.py [hunk 1]
```diff
@@ -163,6 +163,15 @@ def post(self):
         indice_name = json_data["indice_name"]
         metadata_provider = self.clip_resources[indice_name].metadata_provider
         metas = metadata_provider.get(ids, self.clip_resources[indice_name].columns_to_return)
+        for meta in metas:
+            if meta is not None and "image_path" in meta:
```
Comment (rom1504) [inline clip_retrieval/clip_back.py:167]: If confirmed to work, let's put this in a function and reuse in 2 places

**Comment by rom1504** [comment]
Does it fit it for you @D0miH ?

**Comment by D0miH** [comment]
> Does it fit it for you @D0miH ?

Yeah, it works for me. Sorry, forgot to mention that ;)

**Comment by rom1504** [comment]
Ok great, thanks for spotting the issue

**Pull request merged**